pkgs/applications/virtualization/nixpacks/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, clippy }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nixpacks";
+  version = "v0.1.7";
+  doCheck = true;
+  cargoSha256 = "iA8ODh1NmUBfyH6syvVD32jDeLYG2LeLTo2lQoO93lc=";
+  nativeBuildInputs = [ clippy ];
+  src = fetchFromGitHub {
+    owner = "railwayapp";
+    repo = pname;
+    rev = version;
+    sha256 = "UxdK2e5VYcOEYdZn0oGRFIVGiwnPixiZ3rOnqJDSQO8=";
+  };
+  # skip `cargo test` due tests FHS dependency
+  checkPhase = ''
+    runHook preCheck
+    cargo check
+    cargo clippy
+    runHook postCheck
+  '';
+  meta = with lib; {
```

```suggestion
  pname = "nixpacks";
  version = "0.1.7";

  src = fetchFromGitHub {
    owner = "railwayapp";
    repo = pname;
    rev = "v${version}";
    sha256 = "UxdK2e5VYcOEYdZn0oGRFIVGiwnPixiZ3rOnqJDSQO8=";
  };

  cargoSha256 = "iA8ODh1NmUBfyH6syvVD32jDeLYG2LeLTo2lQoO93lc=";

  meta = with lib; {
```
We don't want to run clippy because an update of it should not break packages.